app/controllers/thredded/application_controller.rb

```diff
@@ -88,10 +88,17 @@ def pundit_user
       thredded_current_user
     end
 
+    # Returns the `@messageboard` instance variable.
+    # If `@messageboard` is not set, it first sets it to the topic with the slug or ID given by
+    # `params[:messageboard_id]`.
+    #
+    # @return [Thredded::Messageboard]
+    # @raise [Thredded::Errors::MessageboardNotFound] if the topic with the given slug does not exist.
```

Comment:
> Also why are we using MessageboardNotFound rather than the more normal (but less specific) ActiveRecord::NotFound -- can see it's a style across different models, but wondering what it gives us

The generic exception is fine but there was a time when I wrote this that I wanted a specific error message to show up with regards to a *messageboard* not existing.